vuu-ui/cypress/pages/ShellWithNewTheme.js

```diff
@@ -0,0 +1,31 @@
+import { SHELL_WITH_NEW_THEME_URL } from "../support/e2e/constants";
+
+export class ShellWithNewTheme {
+  visit() {
+    cy.visit(SHELL_WITH_NEW_THEME_URL);
+  }
+
+  getContextMenuButton() {
+    return cy
+      .findByRole("tablist", { name: "tab headers" })
```

Comment:
I see you've added this `aria-label`, but what is "tab headers"? I'm struggling to relate this to the page.

Reply:
I went with just `layouts` in the end (42c04a48). I figured that label - in combination with the existing role ("tab list") - would be enough to specify it uniquely.

Reply:
As discussed, went with `data tabs` (8acc2513)